### ISSUE TITLE
wrong use of classifier Layer for posterior_cls_prob

### ISSUE THREAD

**Issue by NeVerDieKinG** [bdsaglam/torch-scae#15]
in file stacked_capsule_auto_encoder.py line 211

res.posterior_cls_prob = self.prior_classifier(

should be

res.posterior_cls_prob = self.posterior_classifier(

otherwise it use same layer with prior_cls_prob

**Comment by bdsaglam** [comment]
@NeVerDieKinG Thanks for reporting it. I'm not maintaining it anymore but if you'd like to open a PR that's welcome.

**Comment by bdsaglam** [comment]
Fixed in #16: the posterior probe now uses `posterior_classifier`, and both probes train on logits. Thanks for the report.
